t/03-proxy_wasm/sdks/003-assemblyscript_sdk.t

```diff
@@ -0,0 +1,111 @@
+# vim:set ft= ts=4 sts=4 sw=4 et fdm=marker:
+
+use strict;
+use lib '.';
+use t::TestWasm;
+
+skip_valgrind('wasmtime');
+skip_no_nodejs();
+
+plan tests => repeat_each() * (blocks() * 5);
+
+run_tests();
+
+__DATA__
+
+=== TEST 1: proxy_wasm AssemblyScript SDK - addheader example
+--- load_nginx_modules: ngx_http_echo_module
+--- wasm_modules: assemblyscript_addheader
+--- config
+    location /t {
+        proxy_wasm assemblyscript_addheader 'wasm';
+        echo ok;
+    }
+--- response_body
+ok
+--- response_headers
+hello: wasm
+--- no_error_log
+[error]
+[crit]
+
+
+
+=== TEST 2: proxy_wasm AssemblyScript SDK - auth example, authorized
+--- load_nginx_modules: ngx_http_echo_module
+--- wasm_modules: assemblyscript_auth
+--- tcp_listen: 12345
```

Comment:
Can we use the `SERVER_PORT2` variable here instead of hard-coding a port number? Because `gen_rand_port` will actually check for used ports iirc.